config/sc-config.yaml

```diff
@@ -157,9 +157,8 @@ grafana:
     # the correct IP/range and port should be added to .networkPolicies.monitoring.grafana.externalDataSources
     additionalDatasources: {}
     additionalConfigValues: ""
-    # additionalConfigValues: |-
-    #   dataproxy:
-    #     timeout: 600
+    dataproxy:
+      timeout: 600
```

Comment:
Since this was the example value I'm curious if we could deprecate `additionalConfigValues`? It would be nice to not have any unknown options in our config at some point in the future!

Reply:
:thinking: I like the idea of not having any unknown options, but I also like the flexibility of having these :sweat_smile:

Any other thoughts on this?

Reply:
We are using the additionalConfigValues live right now e.g. to disable the angular deprecation notice.

Reply:
Okay, I know we've also used it to increase `dataproxy.timeout` a few times, even though it won't be needed for that exact option anymore I'd say there's reason enough to keep it.

Reply:
Unfortunate. :smile: